templates/default.html

```diff
@@ -13,7 +13,7 @@
     {% if thumbnail %}<meta property="og:image" content="{{ baseUrl }}{{ thumbnail }}" />{% endif %}
 
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:400italic,400,600|Montserrat:400,600">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/solarized-dark.min.css">
+    <link rel="stylesheet" href="/css/highlight/obsidian.css">
```

Comment:
Let's delete the local CSS and just use `//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/obsidian.min.css` (so simply replace the theme name in the previous line)